src/App.js

```diff
@@ -1,36 +1,34 @@
 import React, { Component } from 'react';
 import { Switch, BrowserRouter, Route, Link } from 'react-router-dom';
-
 import HomePage from './components/HomePage/HomePage';
 import AboutUs from './components/AboutUs/AboutUs';
 import Guests from './components/Guests/Guests';
 import Contact from './components/Contact/Contact';
 import Sponsors from './components/Sponsors/Sponsors';
 import User from './components/User/User';
 import LoggedIn from './LoggedIn';		
+import NotFound from './components/NotFound/NotFound'
```

Comment:
change name for NotFound. Not very intutive. You can use 404NotFound

Reply:
`An identifier or keyword cannot immediately follow a numeric literal.`
Can't use 404NotFound, how about Error404NotFound?

Reply:
fine !